cli/internal/turbostate/turbostate.go

```diff
@@ -22,32 +30,70 @@ type LoginPayload struct {
 	SsoTeam string `json:"sso_team"`
 }
 
+// PrunePayload is the extra flags passed for the `prune` subcommand
+type PrunePayload struct {
+	Scope     []string `json:"scope"`
+	Docker    bool     `json:"docker"`
+	OutputDir string   `json:"output_dir"`
+}
+
+// RunPayload is the extra flags passed for the `run` subcommand
+type RunPayload struct {
```

Comment:
I'm feeling a little icky about parts of the codebase depending on magic strings from Rust land, I don't see a good way to avoid it at this time though. Do you think it would be a good idea to try to contain some of the knowledge in this package where it's easier to contain the blast radius?

I'm imagining that in `run.go` instead of having logic around checking if it's a dry run that could live here along with all of the magic strings:
```go
func (r *RunPayload) IsDryRun() (bool, error) {
    if r.DryRun == "" {
        return false, nil
    }
    recognizedValue := r.DryRun == _dryRunTextValue || r.DryRun == _dryRunJSONValue || r.DryRun == _dryRunNoValue
    if !recognizedValue {
        return false, fmt.Errorf("invalid dry-run mode: %v", runPayload.DryRun)
    }
    return true, nil
}
```

Reply:
We could serialize the constants from Rust land to Go land. It'd be a little more overhead but really not a big deal

Reply:
Yeah I'm thinking we start with this solution, and then eventually what I want to do is do all of the args to opts translation in the Rust side so that the constants stay on the Rust side.